src/main/java/com/ericsson/ei/controller/RuleCheckControllerImpl.java

```diff
@@ -104,7 +105,7 @@
         } catch (Exception e) {
             String errorMessage = "Failed to run rule on event. Error message:\n" + e.getMessage();
             LOGGER.error(errorMessage, e);
-            return new ResponseEntity<>(errorMessage, HttpStatus.BAD_REQUEST);
+            return new ResponseEntity<>(ResponseMessage.createJsonMessage(errorMessage), HttpStatus.BAD_REQUEST);
```

Comment:
Here we might need the information from what issue there was since the user inputed something wrong. I hope e.getMessage is enough. Stacktrace is too much.

Reply:
I just left it as is or did you have something else in mind?